File: app/src/main/java/com/example/proyecto/Util/New/RetrofitHelper.java
package com.example.proyecto.Util.New;

import static com.example.proyecto.Util.Util.RUTA;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitHelper {
    public static Retrofit retrofit;
    public static Retrofit getRetrofit(){
        retrofit = new Retrofit.Builder()
                .baseUrl(RUTA)
                .addConverterFactory(GsonConverterFactory.create())
                .build();
        return retrofit;
    }
}

File: app/src/main/java/com/example/proyecto/Util/New/Kotlinclass/RetrofitHelper.kt
package com.example.proyecto.Util.New.Kotlinclass

import com.example.proyecto.Util.New.RetrofitHelper
import com.example.proyecto.Util.Util
import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object RetrofitHelper {
    fun getRetrofit(): Retrofit {
        return Retrofit.Builder()
            .baseUrl(Util.RUTA)
            .addConverterFactory(GsonConverterFactory.create())
            .build()
    }
}